pymilvus_orm/collection.py

```diff
@@ -107,6 +110,7 @@ def __init__(self, name, schema=None, **kwargs):
             0
         """
         self._name = name
+        self.using = using
```

Comment:
Can this variable be renamed to `_using`?

Reply:
ok